src/links/dto/create-link.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsUrl, IsUUID } from 'class-validator';
+
+export class CreateLinkDto {
+  @IsUrl()
+  shortUrl: string;
+
+  @IsUrl()
```

Comment:
**praise**: Good use of `@IsUrl()` to validate `originalUrl`. This helps maintain data integrity.